cli/bzk/service.go

```diff
@@ -90,26 +94,21 @@ func doRestartService() {
 		log.Fatal(err)
 	}
 
-	allContainers, err = client.Ps(&docker.PsOptions{
```

Comment:
If you focus this PR only on the mongoURI part, I can merge it, and refactor the "cache" after that